java/src/com/ibm/streamsx/topology/context/remote/RemoteContext.java

```diff
@@ -16,7 +16,8 @@
     public enum Type {
         TOOLKIT,        
         BUILD_ARCHIVE,
-        REMOTE_BUILD_AND_SUBMIT
+	REMOTE_BUILD_AND_SUBMIT,
```

Comment:
Not sure I understand, I was just expecting a config property behaving like the existing ones. It's in the `Map` passed at submit time.